source/MaterialXRender/ImageHandler.h

```diff
@@ -166,17 +166,13 @@ class ImageHandler
     /// @param image The image to be saved
     /// @param verticalFlip Whether the image should be flipped in Y during save
     /// @return if save succeeded
-    virtual bool saveImage(const FilePath& filePath,
-                           ConstImagePtr image,
-                           bool verticalFlip = false);
+    bool saveImage(const FilePath& filePath, ConstImagePtr image, bool verticalFlip = false);
 
     /// Acquire an image from the cache or file system.  If the image is not
     /// found in the cache, then each image loader will be applied in turn.
     /// @param filePath File path of the image.
-    /// @param generateMipMaps Generate mip maps if supported.
```

Comment:
Noticed that this option has been removed. It would be good to still allow this but perhaps keep this as a option which can be set on the ImageHandler ? We sometimes don't want mipmaps to save on memory. Or perhaps this only applies when binding for hardware usage ?

Reply:
Initially, I thought the same thing, but this flag didn't actually have any effect on the results of ImageHandler::acquireImage.  It's the method ImageHandler::createRenderResources that decides whether mipmaps should be generated, and that method is called independently of ImageHandler::acquireImage.